### PR TITLE
Reac 4 reac 6 arrows

### PR DESCRIPTION
**Pull request by Lukasz-pluszczewski**
Must be merged 3rd

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -104,32 +107,65 @@ export default class Carousel extends Component {
...
-      <button className="BrainhubCarousel__arrows BrainhubCarousel__arrow-left" type="button">Prev</button>
-    );
+    if (this.props.arrowLeft) {
+      return this.renderArrowWithAddedHandler(this.props.arrowLeft, this.prevSlide);
+    }
+    if (this.props.arrows) {
+      return (
+        <div
+          className="BrainhubCarousel__arrows BrainhubCarousel__arrow-left"
+          type="button"
```
Comment (RobertHebel) [inline src/components/Carousel.js]: change to role

File: src/components/Carousel.js [hunk 2]
```diff
@@ -104,32 +107,65 @@ export default class Carousel extends Component {
...
-      <button className="BrainhubCarousel__arrows BrainhubCarousel__arrow-right" type="button">Next</button>
-    );
+    if (this.props.arrowRight) {
+      return this.renderArrowWithAddedHandler(this.props.arrowRight, this.nextSlide);
+    }
+    if (this.props.arrows) {
+      return (
+        <div
+          className="BrainhubCarousel__arrows BrainhubCarousel__arrow-right"
+          type="button"
```
Comment (RobertHebel) [inline src/components/Carousel.js]: same as above

File: src/components/Carousel.js (hunk 1 above)
Comment (Lukasz-pluszczewski) [reply src/components/Carousel.js]: Fixed in future PR

File: src/components/Carousel.js (hunk 2 above)
Comment (Lukasz-pluszczewski) [reply src/components/Carousel.js]: Fixed in future PR

File: src/components/Carousel.js [hunk 3]
```diff
@@ -1,39 +1,190 @@
...
+        </div>
+      );
+    }
+    return null;
+  }
+
+  renderCarouselDots = () => {
+    return (
+      <ul className="BrainhubCarousel__dots">
+        {this.props.children.map((carouselItem, index) => (
```
Comment (adam-golab) [inline src/components/Carousel.js]: Could you move this arrow function to a method?

Comment (Lukasz-pluszczewski) [reply src/components/Carousel.js]: Removed in future PR

**Pull request merged**